cmd/ocp.go

```diff
@@ -89,13 +89,17 @@ func openShiftCmd() *cobra.Command {
 		kubeClientProvider := config.NewKubeClientProvider("", "")
 		wh = workloads.NewWorkloadHelper(workloadConfig, ocpConfig, kubeClientProvider)
 		envVars := map[string]string{
-			"ES_SERVER":     esServer,
-			"ES_INDEX":      esIndex,
-			"QPS":           fmt.Sprintf("%d", QPS),
-			"BURST":         fmt.Sprintf("%d", burst),
-			"GC":            fmt.Sprintf("%v", gc),
-			"GC_METRICS":    fmt.Sprintf("%v", gcMetrics),
-			"INDEXING_TYPE": indexer,
+			"ES_SERVER":      esServer,
+			"ES_INDEX":       esIndex,
+			"LOCAL_INDEXING": fmt.Sprintf("%v", localIndexing),
+			"QPS":            fmt.Sprintf("%d", QPS),
+			"BURST":          fmt.Sprintf("%d", burst),
+			"GC":             fmt.Sprintf("%v", gc),
+			"GC_METRICS":     fmt.Sprintf("%v", gcMetrics),
+			"INDEXING_TYPE":  indexer,
```

Comment:
I think we can remove references to INDEXING_TYPE, and the varible indexer, this variable was created to configure different indexers types in the same indexerConfig but since local and elasticsearch indexers are pre-configured (and not enabled) by default we don't need it

Reply:
I managed to run it w/o INDEXING_TYPE...

- No indexing
```shell
$ ./bin/amd64/kube-burner-ocp cluster-density-v2 --iterations 1 --churn=false
time="2024-04-26 10:09:11" level=info msg="🔥 Starting kube-burner (pull-56@7a62792ff2bd841618f4d74cb5703aec3fa8b821) with UUID 12e5f998-62c3-4dfa-820a-ca70ecd32b9b" file="job.go:99"
time="2024-04-26 10:09:11" level=info msg="📈 Registered measurement: podLatency" file="factory.go:91"                                                                                                                                        
...
time="2024-04-26 10:10:38" level=info msg="👋 Exiting kube-burner 12e5f998-62c3-4dfa-820a-ca70ecd32b9b" file="helpers.go:111"
```


- Local indexing

```shell
$ ./bin/amd64/kube-burner-ocp cluster-density-v2 --iterations 1 --churn=false --local-indexing                                                                                           
time="2024-04-26 10:15:57" level=info msg="📁 Creating local indexer: indexer-0" file="metrics.go:62"                                                                                                                                         
time="2024-04-26 10:15:57" level=info msg="👽 Initializing prometheus client with URL: https://prometheus-k8s-openshift-monitoring.apps.ocp-ci.perfscale.devcluster.openshift.com" file="prometheus.go:47"                                    
time="2024-04-26 10:15:58" level=info msg="📁 Creating local indexer: indexer-1" file="metrics.go:62"                                                                                                                                         
time="2024-04-26 10:15:58" level=info msg="👽 Initializing prometheus client with URL: https://prometheus-k8s-openshift-monitoring.apps.ocp-ci.perfscale.devcluster.openshift.com" file="prometheus.go:47"
time="2024-04-26 10:15:58" level=info msg="🔥 Starting kube-burner (pull-56@7a62792ff2bd841618f4d74cb5703aec3fa8b821) with UUID d0c3698e-2bda-426a-ac62-cc38b27d5db8" file="job.go:99"
time="2024-04-26 10:15:58" level=info msg="📈 Registered measurement: podLatency" file="factory.go:91"
...
time="2024-04-26 10:17:48" level=info msg="Indexing [1] documents from metric max-memory-workers" file="prometheus.go:225"                                                                                                                    
time="2024-04-26 10:17:48" level=info msg="File reporting-metrics/max-memory-workers.json created with 1 documents" file="prometheus.go:230"                                                                                                  
time="2024-04-26 10:17:48" level=info msg="Indexing [0] documents from metric max-memory-sum-infra-start" file="prometheus.go:225"                                                                                                            
time="2024-04-26 10:17:48" level=error msg="Empty document list in max-memory-sum-infra-start" file="prometheus.go:228"                                                                                                                       
time="2024-04-26 10:17:48" level=info msg="Indexing [0] documents from metric memory-cluster-usage-ratio-start" file="prometheus.go:225"                                                                                                      
time="2024-04-26 10:17:48" level=error msg="Empty document list in memory-cluster-usage-ratio-start" file="prometheus.go:228"                                                                                                                 
time="2024-04-26 10:17:48" level=info msg="Finished execution with UUID: d0c3698e-2bda-426a-ac62-cc38b27d5db8" file="job.go:276"                                                                                                              
time="2024-04-26 10:17:48" level=info msg="Garbage collecting jobs" file="job.go:289"                                                                                                                                                         
time="2024-04-26 10:17:48" level=info msg="Indexing cluster metadata document" file="helpers.go:143"                                                                                                                                          
time="2024-04-26 10:17:48" level=info msg="File reporting-metrics/clusterMetadata.json created with 1 documents" file="helpers.go:151"                                                                                                        
time="2024-04-26 10:17:48" level=info msg="Indexing cluster metadata document" file="helpers.go:143"                                                                                                                                          
time="2024-04-26 10:17:48" level=info msg="File reporting-metrics/clusterMetadata.json created with 1 documents" file="helpers.go:151"                                                                                                        
time="2024-04-26 10:17:48" level=info msg="👋 Exiting kube-burner d0c3698e-2bda-426a-ac62-cc38b27d5db8" file="helpers.go:111"                           
```


- os/es indexing
```
$ ./bin/amd64/kube-burner-ocp cluster-density-v2 --iterations=1 --churn=false --es-server ${ES_SERVER} --es-index kube-burner 
time="2024-04-26 11:49:28" level=info msg="📁 Creating opensearch indexer: indexer-0" file="metrics.go:62"
time="2024-04-26 11:49:30" level=info msg="👽 Initializing prometheus client with URL: https://prometheus-k8s-openshift-monitoring.apps.ocp-ci.perfscale.devcluster.openshift.com" file="prometheus.go:47"
time="2024-04-26 11:49:31" level=info msg="🔔 Initializing alert manager for prometheus: https://prometheus-k8s-openshift-monitoring.apps.ocp-ci.perfscale.devcluster.openshift.com" file="alert_manager.go:85"
time="2024-04-26 11:49:31" level=info msg="🔥 Starting kube-burner (pull-56@7a62792ff2bd841618f4d74cb5703aec3fa8b821) with UUID f69dc36f-a86c-4f25-a5bc-7fb5d3649318" file="job.go:99"
time="2024-04-26 11:49:31" level=info msg="📈 Registered measurement: podLatency" file="factory.go:91"
...
time="2024-04-26 11:53:46" level=info msg="Indexing [0] documents from metric cpu-infra-start" file="prometheus.go:225"
time="2024-04-26 11:53:46" level=info msg="Indexing skipped due to 0 docs" file="prometheus.go:230"
time="2024-04-26 11:53:46" level=info msg="Indexing [1] documents from metric cpu-workers" file="prometheus.go:225"
time="2024-04-26 11:53:47" level=info msg="Indexing finished in 442ms: created=1" file="prometheus.go:230"
time="2024-04-26 11:53:47" level=info msg="Finished execution with UUID: f69dc36f-a86c-4f25-a5bc-7fb5d3649318" file="job.go:276"
time="2024-04-26 11:53:47" level=info msg="Garbage collecting jobs" file="job.go:289"
time="2024-04-26 11:53:47" level=info msg="Indexing cluster metadata document" file="helpers.go:143"
time="2024-04-26 11:53:47" level=info msg="Indexing finished in 405ms: created=1" file="helpers.go:151"
time="2024-04-26 11:53:47" level=info msg="👋 Exiting kube-burner f69dc36f-a86c-4f25-a5bc-7fb5d3649318" file="helpers.go:111"
rsevilla@wonderland ~/labs/kube-burner-ocp (pull-56) $ echo $ES_SERVER 

``` 


I've done this changes only for the cd-v2 workload, and pushed a new commit on top of the code of this PR at https://github.com/kube-burner/kube-burner-ocp/commit/820e148178f73533c9dda185aa06d04d19059040

PTAL

Reply:
@rsevilla87 ohh, so do we want to hardcode `opensearch` as default indexer for all the config files?

Reply:
yeah, we're the main users of kube-burner-ocp, we can have some hardcoded stuff w/o issues :)